examples/benchmark.py

```diff
@@ -34,6 +34,9 @@
     parser.add_argument(dest="execmode", nargs="?", default="run",
                         choices=["run", "test", "bench", "plot"],
                         help="Exec modes")
+    parser.add_argument("--bench-mode", "-bm", dest="benchmode", default="all",
+                        choices=["all", "at", "dse"],
+                        help="Choose what to benchmark (all, at, dse).")
```

Comment:
Good spot. Actually, what I think I have to do is to turn this ``at`` into ``blocking``. Because in theory the DSE itself could be autotuned (although at present there should be no need for it). In other words, the confusion here arises from the fact that so far we assume AT==blocking although they two different things